$OpenBSD: patch-Makefile,v 1.1 2010/11/24 13:09:11 jasper Exp $

- Don't hide the compile commands.
- Don't unconditionally strip the binary.

--- Makefile.orig	Wed Nov 24 09:29:08 2010
+++ Makefile	Wed Nov 24 09:29:35 2010
@@ -13,13 +13,13 @@ options:
 
 .c.o:
 	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.h config.mk
 
 dvtm: ${OBJ}
 	@echo CC -o $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
+	${CC} -o $@ ${OBJ} ${LDFLAGS}
 
 debug: clean
 	@make CFLAGS='${DEBUG_CFLAGS}'
@@ -40,8 +40,6 @@ dist: clean
 	@rm -rf dvtm-${VERSION}
 
 install: dvtm
-	@echo stripping executable
-	@strip -s dvtm
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
 	@mkdir -p ${DESTDIR}${PREFIX}/bin
 	@cp -f dvtm ${DESTDIR}${PREFIX}/bin
